### DISCUSSION TITLE
Is there a way to disable the `ExchangeRateConverter` everywhere?

### DISCUSSION THREAD

**Discussion by victorbalssa** [firefly-iii/firefly-iii#8679] [Firefly III] [answered]
I have multiple accounts with 15k tx in total using different currency and since a few month I started to see slowness in certain api requests.

Checking the Laravel debugbar, I see ~13k db queries for `/net-worth` and `/summary/basic`.

Most of the queries are on the table `currency_exchange_rates` hence the question:

- Is there a way to disable the `ExchangeRateConverter` everywhere?

Thanks!

**Comment by JC5** [comment, marked as answer]
It will be in the next release, good idea.

**Reply by victorbalssa** [reply]
@JC5 bumping this, it's taking more than 30s to load with _a lot_ of queries coming from `ExchangeRateConverter`.
Thanks!

**Reply by JC5** [reply]
Thanks for the reminder!
